.github/workflows/python-unit-testing.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Services Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies (Keywords)
+      working-directory: ./services/keywords
```

Comment:
Maybe just make a "Install pytest" step as the first step so you don't have to repeat?

Reply:
Mostly having trouble because every separate app has a different requirements.txt and I need to pip install everything inside them

Reply:
Yeah, that's ok. You could write a small script to find all the requirements.txt and install them, but this is fine.

I would move your `pip install pytest pytest-mock` to its own step at the beginning. Otherwise, if someone switches the order of installing search engine dependencies and keywords dependencies, the unit tests will confusingly break.